README.md

```diff
@@ -1,90 +1,84 @@
+- [Usage](#usage)
+  - [Class Declaration *Optional.*](#class-declaration-optional)
+- [Reference](#reference)
+- [Limitations](#limitations)
+- [Development](#development)
 
 ## Description
 
-puppetlabs-self_service aims to provide a mechnism to alert the end-user when Puppet Enterprise is not in an ideal state.
+puppetlabs-self_service aims to provide a mechanism to alert the end-user when Puppet Enterprise is not in an ideal state.
 It uses pre-set indicators and has a simplified output that directs the end-user to next steps for resolution.
 
 Users of the tool should expect greater ability to provide the self served resolutions, as well as shorter incident resolution times with Puppet Enterprise Support due to higher quality information available to support cases.
 
-
 ## Setup
 
 ### What self_service affects
 
-This module installs a structured fact named self_service, which contains an array of key pairs tha simply output an indicator ID and a boolean value. 
+This module installs a structured fact named self_service, which contains an array of key pairs tha simply output an indicator ID and a boolean value.
 
 ### Setup Requirements
 
 Plugin-Sync should be enabled to deliver the facts this module requires
 
 ### Beginning with self_service
 
-puppetlabs-self_service primarly provides the indicators by means of facts so installing the module and allowing plugin sync to occur will allow the module to start functioning
+puppetlabs-self_service primarily provides the indicators by means of facts so installing the module and allowing plugin sync to occur will allow the module to start functioning
 
 ## Usage
 
-
 The Facts contained in this module can be used for direct consumption by monitoring tools such as Splunk, any element in the structured fact `self_service` reporting as "false" indicates a fault state in Puppet Enterprise.
 When any of the elements reports as false, the incident ID should be looked up in the reference section for next steps
 
-Alternativly assigning the class self_service to the infrastructure  Will "Notify" on Each Puppet run if any of the indicators are in a fault state.
+Alternatively assigning the class self_service to the infrastructure  Will "Notify" on Each Puppet run if any of the indicators are in a fault state.
 
-### Class Delcaration *Optional.*
+### Class Declaration *Optional.*
 
 To activate the notification functions of this module, classify your Puppet Infrastructure  with the self_service class using your preferred classification method. Below is an example using site.pp.
 
-```
+```puppet
 node 'node.example.com' {
   include self_service
 }
 ```
 
-While the entirity of the default indictors should be reported on for maximum coverage, it may be nessary to make exceptions for your particular environment.
+While the entirety of the default indicators should be reported on for maximum coverage, it may be necessarily to make exceptions for your particular environment.
```

Comment:
While the entirety of the default indicators should be reported on for maximum coverage, it may be **necessary** to make exceptions for your particular environment.